servers/physics_2d/godot_physics_server_2d.cpp

```diff
@@ -1044,40 +1044,67 @@ void GodotPhysicsServer2D::joint_clear(RID p_joint) {
 
 void GodotPhysicsServer2D::joint_set_param(RID p_joint, JointParam p_param, real_t p_value) {
 	GodotJoint2D *joint = joint_owner.get_or_null(p_joint);
-	ERR_FAIL_COND(!joint);
+	ERR_FAIL_COND(joint);
 
 	switch (p_param) {
-		case JOINT_PARAM_BIAS:
+		case JOINT_PARAM_BIAS: {
 			joint->set_bias(p_value);
-			break;
-		case JOINT_PARAM_MAX_BIAS:
+		} break;
+		case JOINT_PARAM_MAX_BIAS: {
 			joint->set_max_bias(p_value);
-			break;
-		case JOINT_PARAM_MAX_FORCE:
+		} break;
+		case JOINT_PARAM_MAX_FORCE: {
 			joint->set_max_force(p_value);
-			break;
+		} break;
+		case JOINT_PARAM_BREAK_FORCE: {
+			joint->set_break_force(p_value);
+		} break;
+		case JOINT_PARAM_BREAK_TORQUE: {
+			joint->set_break_torque(p_value);
+		} break;
 	}
 }
 
 real_t GodotPhysicsServer2D::joint_get_param(RID p_joint, JointParam p_param) const {
 	const GodotJoint2D *joint = joint_owner.get_or_null(p_joint);
-	ERR_FAIL_COND_V(!joint, -1);
+	ERR_FAIL_NULL_V(joint, -1);
 
 	switch (p_param) {
-		case JOINT_PARAM_BIAS:
+		case JOINT_PARAM_BIAS: {
 			return joint->get_bias();
-			break;
-		case JOINT_PARAM_MAX_BIAS:
+		} break;
+		case JOINT_PARAM_MAX_BIAS: {
 			return joint->get_max_bias();
-			break;
-		case JOINT_PARAM_MAX_FORCE:
+		} break;
+		case JOINT_PARAM_MAX_FORCE: {
 			return joint->get_max_force();
-			break;
+		} break;
+		case JOINT_PARAM_BREAK_FORCE: {
+			return joint->get_break_force();
+		} break;
+		case JOINT_PARAM_BREAK_TORQUE: {
+			return joint->get_break_torque();
+		} break;
 	}
 
 	return 0;
 }
 
+real_t GodotPhysicsServer2D::joint_get_state(RID p_joint, JointState p_param) const {
+	const GodotJoint2D *joint = joint_owner.get_or_null(p_joint);
+	ERR_FAIL_NULL_V(joint, -1);
+
+	switch (p_param) {
+		case JOINT_STATE_REACTION_FORCE:
+			return joint->get_reaction_force();
+			break;
+		case JOINT_STATE_REACTION_TORQUE:
+			return joint->get_reaction_torque();
+			break;
```

Comment:
```suggestion
		case JOINT_STATE_REACTION_TORQUE:
			return joint->get_reaction_torque();
```
No need to add `break` when returning